src/hooks/use-format-localized-timestamps.tsx

```diff
@@ -0,0 +1,41 @@
+import { Duration, intervalToDuration } from 'date-fns';
+import { useCallback } from 'react';
+import { formatDistance } from '../lib/date';
```

Comment:
This function seems to be used in conjunction with `date-fns`'s `formatDuration` function when looking at `src/hooks/use-expiration-date.ts`.

Should we do something similar here? I also see that date-fns has a built-in `formatDistance` function ([docs](https://date-fns.org/v4.1.0/docs/formatDistance)). I'm not familiar with that one, though—maybe there's a good reason for us not to use it.

Reply:
I think we use the `format` in `use-expiration-date.ts` to display a specific format that contains `days and hours`, and `hours and minutes` together in the same sentence. Not sure if we need a specific format in this case, or if it's ok using the default format.

I tested it and it seems to be working good.

<img width="254" alt="Screenshot 2024-12-05 at 11 18 26" src="https://github.com/user-attachments/assets/8c9f19da-c0a2-4117-9ed3-634dd26f04f5">